common/src/main/java/com/alibaba/nacos/common/executor/ExecutorFactory.java

```diff
@@ -27,70 +27,169 @@
 public final class ExecutorFactory {
 
-    private static final ThreadPoolManager THREAD_POOL_MANAGER = ThreadPoolManager.getInstance();
+    public static ExecutorService newSingleExecutorService() {
+        ExecutorService executorService = Executors.newFixedThreadPool(1);
+        return executorService;
+    }
+
+    public static ExecutorService newSingleExecutorService(final ThreadFactory threadFactory) {
+        ExecutorService executorService = Executors.newFixedThreadPool(1, threadFactory);
+        return executorService;
+    }
 
-    public static final String DEFAULT_NAMESPACE = "nacos";
+    public static ExecutorService newFixExecutorService(final int nThreads) {
+        ExecutorService executorService = Executors.newFixedThreadPool(nThreads);
+        return executorService;
+    }
 
+    public static ExecutorService newFixExecutorService(final int nThreads,
+                                                        final ThreadFactory threadFactory) {
+        ExecutorService executorService = Executors.newFixedThreadPool(nThreads, threadFactory);
+        return executorService;
+    }
+
+    public static ScheduledExecutorService newSingleScheduledExecutorService(final ThreadFactory threadFactory) {
+        ScheduledExecutorService executorService = Executors.newScheduledThreadPool(1, threadFactory);
+        return executorService;
+    }
+
+    public static ScheduledExecutorService newScheduledExecutorService( final int nThreads,
+                                                                        final ThreadFactory threadFactory) {
+        ScheduledExecutorService executorService = Executors.newScheduledThreadPool(nThreads, threadFactory);
+        return executorService;
+    }
+
+    public static ThreadPoolExecutor newCustomerThreadExecutor(
+        final int coreThreads,
+        final int maxThreads,
+        final long keepAliveTimeMs,
+        final ThreadFactory threadFactory) {
+        ThreadPoolExecutor executor = new ThreadPoolExecutor(coreThreads, maxThreads,
+            keepAliveTimeMs, TimeUnit.MILLISECONDS,
+            new LinkedBlockingQueue<Runnable>(),
+            threadFactory);
+        return executor;
+    }
+    //TODO remove Deprecated function after replace all module
+    @Deprecated
     public static ExecutorService newSingleExecutorService(final String group) {
         ExecutorService executorService = Executors.newFixedThreadPool(1);
-        THREAD_POOL_MANAGER.register(DEFAULT_NAMESPACE, group, executorService);
         return executorService;
     }
-
+    @Deprecated
     public static ExecutorService newSingleExecutorService(final String group,
-                                                           final ThreadFactory threadFactory) {
+        final ThreadFactory threadFactory) {
         ExecutorService executorService = Executors.newFixedThreadPool(1, threadFactory);
-        THREAD_POOL_MANAGER.register(DEFAULT_NAMESPACE, group, executorService);
```

Comment:
don't change the old methods.
remove them after change to use new mehtod will be ok.

Reply:
ok, i will rollback the old methods now,and remove them after replace all module